package.json

```diff
@@ -108,5 +108,8 @@
     "yargs": "^13.2.2",
     "zone.js": "^0.9.1"
   },
-  "keywords": []
+  "keywords": [],
+  "dependencies": {
+    "handlebars": "^4.3.1"
+  }
```

Comment:
Why we need to add this here?

Reply:
This is the package in question that was causing the error, so this ensures that we have at least this version.

Reply:
Error being the audit issue.

Reply:
Or maybe why it is located in `dependencies` and not `devDependencies`?

Reply:
Or maybe some of the other stuff from `devDependencies` should also be located in `dependencies` instead.

Reply:
tbh, not sure. This was done automatically by Yarn. All upgrades done here
were via their CLI.

Reply:
Okay then, I'm okay with this, but we should maybe file an issue to look into this later.

Reply:
sounds good, I'll do that right now